KryptografiaStosowana/BlockChainP2P.P2PNetwork.Api.Manager/ServiceHelpers/ServiceExtensions.cs

```diff
@@ -16,4 +16,11 @@ public static IServiceCollection AddP2PManagers(this IServiceCollection services
             .AddTransient<IPeerManager, PeerManager>();
         return services;
     }
+
+    public static IServiceCollection AddBlockChainManagers(this IServiceCollection services)
```

Comment:
Teraz sobie myślę, że można by to przerzucić po prostu do jednej metody AddP2PTransientManagers bo mamy jedną solucję menadżerską i później tylko ją wywołać w program.cs